config/requirements-dev.txt

```diff
@@ -1,5 +1,4 @@
 # This is for dev purpose
 -r requirements.txt
-mock==3.0.5
```

Comment:
```
======================================================================
ERROR: Failure: ModuleNotFoundError (No module named 'mock')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/circleci/repo/env/lib/python3.7/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/home/circleci/repo/env/lib/python3.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/circleci/repo/env/lib/python3.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/circleci/repo/env/lib/python3.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/usr/local/lib/python3.7/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/usr/local/lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/circleci/repo/tests/unit/test_helpers.py", line 10, in <module>
    import mock
ModuleNotFoundError: No module named 'mock'
```

Removing mock. good! 
but that means that you need to change the way the import are done.

It now lives in https://docs.python.org/3/library/unittest.mock.html

so for example, you will need to things like this:

```diff
diff --git a/tests/unit/test_urls.py b/tests/unit/test_urls.py
index 7371117c..7d4c7a33 100644
--- a/tests/unit/test_urls.py
+++ b/tests/unit/test_urls.py
@@ -10,8 +10,7 @@ import os.path
 import re
 import sys
 import unittest
-
-from mock import patch
+from unittest.mock import patch
 
 # Add webcompat module to import path
 sys.path.append(os.path.realpath(os.pardir))
```


also because it's not any more an imported vendor module, but a library module, you need to move it up in the first block of import.